images/imaging.py

```diff
@@ -533,20 +560,35 @@ def render_galaxy_image(
         x = particles.x_mbp / data.metadata.a
         y = particles.y_mbp / data.metadata.a
         z = particles.z_mbp / data.metadata.a
-        r = particles.r_200crit / data.metadata.a
+        if "COLIBRE" in ptype:
```

Comment:
```
r_factor = 0.1 if <colibre> else 1.0
r = r_factor * particles.r_200crit / data.metadata.a
radius_name = f"{r_factor if r_factor != 1.0 else ''} $R_{{200, \\rm{{crit}}}}$"
```

Then `radius_name` can be used in `galaxy_attributes` and we can avoid checking for `COLIBRE` again.